### PR TITLE
Renamed tenant admin partials

### PR DESCRIPTION
**Pull request by Coenego**

### CODE REVIEW

**Comment by Coenego** [comment]
Assigning to @bp323 for review

File: apps/timetable/admin/app/js/index.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ define(['gh.core', 'gh.constants', 'jquery-autosuggest', 'validator'], function(
      * @private
      */
     var renderNavigation = function() {
-        gh.utils.renderTemplate('tenant-admin-navigation', {
+        gh.utils.renderTemplate('admin-app-navigation', {
```
Comment (bp323) [inline apps/timetable/admin/app/js/index.js]: Should rename the file as well.

File: apps/timetable/admin/app/js/index.js [hunk 2]
```diff
@@ -72,7 +72,7 @@ define(['gh.core', 'gh.constants', 'jquery-autosuggest', 'validator'], function(
      * @private
      */
     var renderAppUserSearch = function() {
-        gh.utils.renderTemplate('tenant-admin-app-user-search', null, $('#gh-main'), function() {
+        gh.utils.renderTemplate('admin-app-user-search', null, $('#gh-main'), function() {
```
Comment (bp323) [inline apps/timetable/admin/app/js/index.js]: Same here.

File: apps/timetable/admin/app/js/index.js [hunk 3]
```diff
@@ -87,7 +87,7 @@ define(['gh.core', 'gh.constants', 'jquery-autosuggest', 'validator'], function(
      * @private
      */
     var renderUser = function(triposData, user, memberships) {
-        gh.utils.renderTemplate('tenant-admin-app-user', {
+        gh.utils.renderTemplate('admin-app-user', {
```
Comment (bp323) [inline apps/timetable/admin/app/js/index.js]: Same here

File: apps/timetable/admin/app/js/index.js [hunk 4]
```diff
@@ -101,7 +101,7 @@ define(['gh.core', 'gh.constants', 'jquery-autosuggest', 'validator'], function(
      * @private
      */
     var renderApp = function() {
-        gh.utils.renderTemplate('tenant-admin-app', {
+        gh.utils.renderTemplate('admin-app', {
```
Comment (bp323) [inline apps/timetable/admin/app/js/index.js]: Same here

File: apps/timetable/admin/app/js/index.js [hunk 5]
```diff
@@ -113,7 +113,7 @@ define(['gh.core', 'gh.constants', 'jquery-autosuggest', 'validator'], function(
      * @private
      */
     var renderConfig = function(config) {
-        gh.utils.renderTemplate('tenant-admin-config', {
+        gh.utils.renderTemplate('admin-app-config', {
```
Comment (bp323) [inline apps/timetable/admin/app/js/index.js]: Same here. Basically all files that have a filename that differs from the one we use to pass into the template functionality need to be updated.

**Comment by bp323** [comment]
Over to @Coenego for followup

**Comment by Coenego** [comment]
Back to @bp323 for review after follow-up

**Comment by bp323** [comment]
Verified and merging.

**Pull request merged**